frontend/src/pages/ErrorsV2/ErrorsV2.tsx

```diff
@@ -129,6 +145,28 @@ const ErrorsV2: React.FC<React.PropsWithChildren<{ integrated: boolean }>> = ({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [error_secure_id])
 
+	useEffect(() => {
+		if (logCursor) {
+			if (errorObjectForLogLoading) {
+				setLoadingState(AppLoadingState.LOADING)
+			} else {
+				setLoadingState(AppLoadingState.LOADED)
+				console.log('vadim', errorObjectForLogData)
+				if (errorObjectForLogData?.error_object_for_log) {
+					navigate(
```

Comment:
Can this be done at the routing level? Or at least in the parent component of this. Doing this in a `useEffect` seems like it's going to have unintended side effects especially since we're using `navigate`

Reply:
What do you think about adding a new route + component for handling this logic? e.g. add a new route like `/errors/logs/:cursor` that renders a new `<ErrorsLogsRedirect />` component which contains the logic for looking up the error and handling the redirect.

Reply:
i like that idea, certainly feels cleaner to leverage react router. the original reason to go with qs params was the thought that the url pattern of `/errors?log=<cursor>` looks nicer than `/errors/logs/:cursor`, but i don't have a strong preference. cc @jay-khatri

Reply:
updated!